fabfile/db/__init__.py

```diff
@@ -171,15 +210,25 @@ def restore_db_postgres(self, suffix, *args, **kwargs):
         with lcd(self.base_path), settings(hide('warnings'), warn_only=True):
             if not local('ls %s/backups/backup.postgres.%s' % (self.base_path, suffix)).succeeded:
                 return self.list_backups('postgres')
-            local('dropdb %s --username=%s -w ' % (self.db_cfg['NAME'], self.db_cfg['USER']))
+            local('dropdb %s_temp --username=%s -w ' % (self.db_cfg['NAME'],
```

Comment:
VF: Also - we need arguments to pass user/password for restoration
VF: process, because database user from django config could have no
VF: permissions for database-level operations.